Synthesis failed for managed_identities

Hello! Autosynth couldn't regenerate managed_identities. :broken_heart:

Please investigate and fix this issue within 5 business days.  While it remains broken,
this library cannot be updated with changes to the managed_identities API, and the library grows
stale.

See https://github.com/googleapis/synthtool/blob/master/autosynth/TroubleShooting.md
for trouble shooting tips.

Here's the output from running `synth.py`:

```
t 810cad37a1d22def3b8e0e2a5593e7a551477058
2021-04-01 03:56:34,821 autosynth [DEBUG] > Running: git log -1 --pretty=%at 5e16a211ee3f60c681cbd66c41fcc1c26dc67ede
2021-04-01 03:56:34,824 autosynth [DEBUG] > Running: git log -1 --pretty=%at 639f01a6bcb206eb0e621a3f4cb5b1aff9b012aa
2021-04-01 03:56:34,828 autosynth [DEBUG] > Running: git log -1 --pretty=%at 4cadede71cca2e12e07f760bb9d1fb31b1cf8a0b
2021-04-01 03:56:34,831 autosynth [DEBUG] > Running: git log -1 --pretty=%at 8906bd88f51a8a0264bbe66a747a228309a2abec
2021-04-01 03:56:34,835 autosynth [DEBUG] > Running: git log -1 --pretty=%at 969808fddcc8a0da8dcefd1f549bfa5135d25501
2021-04-01 03:56:34,838 autosynth [DEBUG] > Running: git log -1 --pretty=%at 8be76ab1a98eb62d84b4eb8921ddc8897d3c282d
2021-04-01 03:56:34,842 autosynth [DEBUG] > Running: git log -1 --pretty=%at 9a51c9578ff78732058480033ec0bd773612282d
2021-04-01 03:56:34,846 autosynth [DEBUG] > Running: git log -1 --pretty=%at 35dc699a4cb058f1c4608d0dd27168e98e794050
2021-04-01 03:56:34,849 autosynth [DEBUG] > Running: git log -1 --pretty=%at e486a188e9335dff624eb4155ec57b033193d3ce
2021-04-01 03:56:34,853 autosynth [DEBUG] > Running: git log -1 --pretty=%at 1f9bbda67348765de98e458c1ad003aa52d6e4d0
2021-04-01 03:56:34,857 autosynth [DEBUG] > Running: git log -1 --pretty=%at bf845b2deb2c6c2f4961d10af72423a40f429842
2021-04-01 03:56:34,861 autosynth [DEBUG] > Running: git log -1 --pretty=%at cbf8e95fbd0cba135d997762044f6ed5725045c0
2021-04-01 03:56:34,864 autosynth [DEBUG] > Running: git log -1 --pretty=%at f405c718d60484124808adb7fb5963974d654bb4
2021-04-01 03:56:34,868 autosynth [DEBUG] > Running: git log -1 --pretty=%at 3c22d0e8fd123b47bf2142eef713c1dbe8ea6db4
2021-04-01 03:56:34,872 autosynth [DEBUG] > Running: git log -1 --pretty=%at ce4fa26cdadda59de4895a501bf43afecf6adcdf
2021-04-01 03:56:34,875 autosynth [DEBUG] > Running: git log -1 --pretty=%at 355a80d7efb196482e07c8c8ec249b0fdf7d3ff3
2021-04-01 03:56:34,879 autosynth [DEBUG] > Running: git log -1 --pretty=%at 381e8814b304a53c19fbacbece1611378dc8efa0
2021-04-01 03:56:34,883 autosynth [DEBUG] > Running: git log -1 --pretty=%at d370911b4fdb000723de6ffa9759d6efcea1b2e3
2021-04-01 03:56:34,887 autosynth [DEBUG] > Running: git log -1 --pretty=%at 6d0a96c8780aacef9df1b2ef71d28cf5f2938990
2021-04-01 03:56:34,890 autosynth [DEBUG] > Running: git log -1 --pretty=%at c3918feb2415878ac428d728fb473ed4187b7819
2021-04-01 03:56:34,894 autosynth [DEBUG] > Running: git log -1 --pretty=%at 6ce40ff8faf68226782f507ca6b2d497a77044de
2021-04-01 03:56:34,897 autosynth [DEBUG] > Running: git log -1 --pretty=%at 6c7693680fe320b4705dd8500c255843749eb58a
2021-04-01 03:56:34,901 autosynth [DEBUG] > Running: git log -1 --pretty=%at c0507a2bf28ea9f9af82559f2c876d2af598d6d9
2021-04-01 03:56:34,904 autosynth [DEBUG] > Running: git log -1 --pretty=%at 45445f476c39de13be89a3c5a45b67c7a2320fcf
2021-04-01 03:56:34,907 autosynth [DEBUG] > Running: git log -1 --pretty=%at a1ab4d44db02d59ff58810c6d4182d84e4b9abaa
2021-04-01 03:56:34,911 autosynth [DEBUG] > Running: git log -1 --pretty=%at 3a8abe551ea1604a528922d16a9799e4c5660ea7
2021-04-01 03:56:34,914 autosynth [DEBUG] > Running: git log -1 --pretty=%at c539b9b08b3366ee00c0ec1950f4df711552a269
2021-04-01 03:56:34,917 autosynth [DEBUG] > Running: git log -1 --pretty=%at 7c8d16188e68347aac0053a40ab1dc2056a44899
2021-04-01 03:56:34,921 autosynth [DEBUG] > Running: git log -1 --pretty=%at 8586c62be9ba602c65d727485c828ed53ce11960
2021-04-01 03:56:34,924 autosynth [DEBUG] > Running: git log -1 --pretty=%at d6642a0d79ab1500f342e6d2a65b3d44a97841bc
2021-04-01 03:56:34,928 autosynth [DEBUG] > Running: git log -1 --pretty=%at ef705a6d9fd6dcd8f0b13e012cb284332734785b
2021-04-01 03:56:34,931 autosynth [DEBUG] > Running: git log -1 --pretty=%at 6823bdc3be4e744450a8bf04803189c6e7585360
2021-04-01 03:56:34,934 autosynth [DEBUG] > Running: git log -1 --pretty=%at 95dd24960cf9f794ef583e59ad9f1fabe1c4a924
2021-04-01 03:56:34,938 autosynth [DEBUG] > Running: git log -1 --pretty=%at 915925089600094e72e4bfa8cf586c170e6b7109
2021-04-01 03:56:34,942 autosynth [DEBUG] > Running: git checkout 20553dec5dead22330e214312be2f1583025a61a
Note: switching to '20553dec5dead22330e214312be2f1583025a61a'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by switching back to a branch.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -c with the switch command. Example:

  git switch -c <new-branch-name>

Or undo this operation with:

  git switch -

Turn off this advice by setting config variable advice.detachedHead to false

HEAD is now at 20553dec5 chore: Add ml-apis team to codeowners for ML-related clients (#11088)
2021-04-01 03:56:34,981 autosynth [DEBUG] > Running: git checkout 915925089600094e72e4bfa8cf586c170e6b7109
Note: switching to '915925089600094e72e4bfa8cf586c170e6b7109'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by switching back to a branch.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -c with the switch command. Example:

  git switch -c <new-branch-name>

Or undo this operation with:

  git switch -

Turn off this advice by setting config variable advice.detachedHead to false

HEAD is now at 91592508 chore: migrate devtools/build to the PHP microgenerator
2021-04-01 03:56:34,997 autosynth [DEBUG] > Running: git branch -f autosynth-managed_identities-90
2021-04-01 03:56:35,001 autosynth [DEBUG] > Running: git checkout autosynth-managed_identities-90
Switched to branch 'autosynth-managed_identities-90'
2021-04-01 03:56:35,042 autosynth [INFO] > Running synthtool
2021-04-01 03:56:35,042 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2021-04-01 03:56:35,042 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/google-cloud-managed_identities/90/sponge_log.log
2021-04-01 03:56:35,045 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2021-04-01 03:56:35,285 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/google-cloud-ruby/google-cloud-managed_identities/synth.py.
On branch autosynth-managed_identities-90
nothing to commit, working tree clean
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.1/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.1/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 674, in exec_module
  File "<frozen importlib._bootstrap_external>", line 781, in get_code
  File "<frozen importlib._bootstrap_external>", line 741, in source_to_code
  File "<frozen importlib._bootstrap>", line 205, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/google-cloud-ruby/google-cloud-managed_identities/synth.py", line 32
    "ruby-cloud-wrapper-of": "v1:0.3,
                                    ^
SyntaxError: EOL while scanning string literal
2021-04-01 03:56:35,417 autosynth [ERROR] > Synthesis failed
2021-04-01 03:56:35,417 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 20553dec5 chore: Add ml-apis team to codeowners for ML-related clients (#11088)
2021-04-01 03:56:35,476 autosynth [DEBUG] > Running: git checkout autosynth-managed_identities
Switched to branch 'autosynth-managed_identities'
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.1/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.1/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 356, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 191, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 336, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 68, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.1/lib/python3.6/subprocess.py", line 369, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/caea8286-4c61-4f2a-b9b7-eaa5e1e339ce/targets/github%2Fsynthtool;config=default/tests;query=google-cloud-ruby;failed=false).

Autosynth is still having trouble generating managed_identities. :sob:

Please investigate and fix this issue within 5 business days.  While it remains broken,
this library cannot be updated with changes to the managed_identities API, and the library grows
stale.

See https://github.com/googleapis/synthtool/blob/master/autosynth/TroubleShooting.md
for trouble shooting tips.

Here's the output from running `synth.py`:

```
012aa
2021-04-02 03:55:10,904 autosynth [DEBUG] > Running: git log -1 --pretty=%at 4cadede71cca2e12e07f760bb9d1fb31b1cf8a0b
2021-04-02 03:55:10,907 autosynth [DEBUG] > Running: git log -1 --pretty=%at 8906bd88f51a8a0264bbe66a747a228309a2abec
2021-04-02 03:55:10,910 autosynth [DEBUG] > Running: git log -1 --pretty=%at 969808fddcc8a0da8dcefd1f549bfa5135d25501
2021-04-02 03:55:10,913 autosynth [DEBUG] > Running: git log -1 --pretty=%at 8be76ab1a98eb62d84b4eb8921ddc8897d3c282d
2021-04-02 03:55:10,916 autosynth [DEBUG] > Running: git log -1 --pretty=%at 9a51c9578ff78732058480033ec0bd773612282d
2021-04-02 03:55:10,920 autosynth [DEBUG] > Running: git log -1 --pretty=%at 35dc699a4cb058f1c4608d0dd27168e98e794050
2021-04-02 03:55:10,923 autosynth [DEBUG] > Running: git log -1 --pretty=%at e486a188e9335dff624eb4155ec57b033193d3ce
2021-04-02 03:55:10,926 autosynth [DEBUG] > Running: git log -1 --pretty=%at 1f9bbda67348765de98e458c1ad003aa52d6e4d0
2021-04-02 03:55:10,929 autosynth [DEBUG] > Running: git log -1 --pretty=%at bf845b2deb2c6c2f4961d10af72423a40f429842
2021-04-02 03:55:10,932 autosynth [DEBUG] > Running: git log -1 --pretty=%at cbf8e95fbd0cba135d997762044f6ed5725045c0
2021-04-02 03:55:10,935 autosynth [DEBUG] > Running: git log -1 --pretty=%at f405c718d60484124808adb7fb5963974d654bb4
2021-04-02 03:55:10,939 autosynth [DEBUG] > Running: git log -1 --pretty=%at 3c22d0e8fd123b47bf2142eef713c1dbe8ea6db4
2021-04-02 03:55:10,942 autosynth [DEBUG] > Running: git log -1 --pretty=%at ce4fa26cdadda59de4895a501bf43afecf6adcdf
2021-04-02 03:55:10,945 autosynth [DEBUG] > Running: git log -1 --pretty=%at 355a80d7efb196482e07c8c8ec249b0fdf7d3ff3
2021-04-02 03:55:10,948 autosynth [DEBUG] > Running: git log -1 --pretty=%at 381e8814b304a53c19fbacbece1611378dc8efa0
2021-04-02 03:55:10,952 autosynth [DEBUG] > Running: git log -1 --pretty=%at d370911b4fdb000723de6ffa9759d6efcea1b2e3
2021-04-02 03:55:10,955 autosynth [DEBUG] > Running: git log -1 --pretty=%at 6d0a96c8780aacef9df1b2ef71d28cf5f2938990
2021-04-02 03:55:10,958 autosynth [DEBUG] > Running: git log -1 --pretty=%at c3918feb2415878ac428d728fb473ed4187b7819
2021-04-02 03:55:10,961 autosynth [DEBUG] > Running: git log -1 --pretty=%at 6ce40ff8faf68226782f507ca6b2d497a77044de
2021-04-02 03:55:10,965 autosynth [DEBUG] > Running: git log -1 --pretty=%at 6c7693680fe320b4705dd8500c255843749eb58a
2021-04-02 03:55:10,968 autosynth [DEBUG] > Running: git log -1 --pretty=%at c0507a2bf28ea9f9af82559f2c876d2af598d6d9
2021-04-02 03:55:10,971 autosynth [DEBUG] > Running: git log -1 --pretty=%at 45445f476c39de13be89a3c5a45b67c7a2320fcf
2021-04-02 03:55:10,975 autosynth [DEBUG] > Running: git log -1 --pretty=%at a1ab4d44db02d59ff58810c6d4182d84e4b9abaa
2021-04-02 03:55:10,978 autosynth [DEBUG] > Running: git log -1 --pretty=%at 3a8abe551ea1604a528922d16a9799e4c5660ea7
2021-04-02 03:55:10,981 autosynth [DEBUG] > Running: git log -1 --pretty=%at c539b9b08b3366ee00c0ec1950f4df711552a269
2021-04-02 03:55:10,984 autosynth [DEBUG] > Running: git log -1 --pretty=%at 7c8d16188e68347aac0053a40ab1dc2056a44899
2021-04-02 03:55:10,987 autosynth [DEBUG] > Running: git log -1 --pretty=%at 8586c62be9ba602c65d727485c828ed53ce11960
2021-04-02 03:55:10,991 autosynth [DEBUG] > Running: git log -1 --pretty=%at d6642a0d79ab1500f342e6d2a65b3d44a97841bc
2021-04-02 03:55:10,994 autosynth [DEBUG] > Running: git log -1 --pretty=%at ef705a6d9fd6dcd8f0b13e012cb284332734785b
2021-04-02 03:55:10,997 autosynth [DEBUG] > Running: git log -1 --pretty=%at 6823bdc3be4e744450a8bf04803189c6e7585360
2021-04-02 03:55:11,000 autosynth [DEBUG] > Running: git log -1 --pretty=%at 95dd24960cf9f794ef583e59ad9f1fabe1c4a924
2021-04-02 03:55:11,003 autosynth [DEBUG] > Running: git log -1 --pretty=%at 915925089600094e72e4bfa8cf586c170e6b7109
2021-04-02 03:55:11,006 autosynth [DEBUG] > Running: git log -1 --pretty=%at f6b491910e5ebbb7f3bb9b13f53294b6e94b2f48
2021-04-02 03:55:11,009 autosynth [DEBUG] > Running: git log -1 --pretty=%at 859cbd3d49a24a3e8173728ed8de1894062b5655
2021-04-02 03:55:11,013 autosynth [DEBUG] > Running: git log -1 --pretty=%at 56fc6d43fed71188d7e18f3ca003544646c4ab35
2021-04-02 03:55:11,016 autosynth [DEBUG] > Running: git checkout 6b6f501ad33449bf39de72e0c3629d13530dd685
Note: switching to '6b6f501ad33449bf39de72e0c3629d13530dd685'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by switching back to a branch.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -c with the switch command. Example:

  git switch -c <new-branch-name>

Or undo this operation with:

  git switch -

Turn off this advice by setting config variable advice.detachedHead to false

HEAD is now at 6b6f501ad chore(vision): Remove unsupported samples
2021-04-02 03:55:11,056 autosynth [DEBUG] > Running: git checkout 56fc6d43fed71188d7e18f3ca003544646c4ab35
Note: switching to '56fc6d43fed71188d7e18f3ca003544646c4ab35'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by switching back to a branch.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -c with the switch command. Example:

  git switch -c <new-branch-name>

Or undo this operation with:

  git switch -

Turn off this advice by setting config variable advice.detachedHead to false

HEAD is now at 56fc6d43 feat: publish LocationMetadata
2021-04-02 03:55:11,071 autosynth [DEBUG] > Running: git branch -f autosynth-managed_identities-93
2021-04-02 03:55:11,075 autosynth [DEBUG] > Running: git checkout autosynth-managed_identities-93
Switched to branch 'autosynth-managed_identities-93'
2021-04-02 03:55:11,113 autosynth [INFO] > Running synthtool
2021-04-02 03:55:11,114 autosynth [INFO] > ['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']
2021-04-02 03:55:11,114 autosynth [DEBUG] > log_file_path: /tmpfs/src/logs/google-cloud-managed_identities/93/sponge_log.log
2021-04-02 03:55:11,116 autosynth [DEBUG] > Running: /tmpfs/src/github/synthtool/env/bin/python3 -m synthtool --metadata synth.metadata synth.py --
2021-04-02 03:55:11,344 synthtool [DEBUG] > Executing /home/kbuilder/.cache/synthtool/google-cloud-ruby/google-cloud-managed_identities/synth.py.
On branch autosynth-managed_identities-93
nothing to commit, working tree clean
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.1/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.1/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 102, in <module>
    main()
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/tmpfs/src/github/synthtool/env/lib/python3.6/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/tmpfs/src/github/synthtool/synthtool/__main__.py", line 94, in main
    spec.loader.exec_module(synth_module)  # type: ignore
  File "<frozen importlib._bootstrap_external>", line 674, in exec_module
  File "<frozen importlib._bootstrap_external>", line 781, in get_code
  File "<frozen importlib._bootstrap_external>", line 741, in source_to_code
  File "<frozen importlib._bootstrap>", line 205, in _call_with_frames_removed
  File "/home/kbuilder/.cache/synthtool/google-cloud-ruby/google-cloud-managed_identities/synth.py", line 32
    "ruby-cloud-wrapper-of": "v1:0.3,
                                    ^
SyntaxError: EOL while scanning string literal
2021-04-02 03:55:11,466 autosynth [ERROR] > Synthesis failed
2021-04-02 03:55:11,466 autosynth [DEBUG] > Running: git reset --hard HEAD
HEAD is now at 6b6f501ad chore(vision): Remove unsupported samples
2021-04-02 03:55:11,524 autosynth [DEBUG] > Running: git checkout autosynth-managed_identities
Switched to branch 'autosynth-managed_identities'
Traceback (most recent call last):
  File "/home/kbuilder/.pyenv/versions/3.6.1/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kbuilder/.pyenv/versions/3.6.1/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 356, in <module>
    main()
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 191, in main
    return _inner_main(temp_dir)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 336, in _inner_main
    commit_count = synthesize_loop(x, multiple_prs, change_pusher, synthesizer)
  File "/tmpfs/src/github/synthtool/autosynth/synth.py", line 68, in synthesize_loop
    has_changes = toolbox.synthesize_version_in_new_branch(synthesizer, youngest)
  File "/tmpfs/src/github/synthtool/autosynth/synth_toolbox.py", line 259, in synthesize_version_in_new_branch
    synthesizer.synthesize(synth_log_path, self.environ)
  File "/tmpfs/src/github/synthtool/autosynth/synthesizer.py", line 120, in synthesize
    synth_proc.check_returncode()  # Raise an exception.
  File "/home/kbuilder/.pyenv/versions/3.6.1/lib/python3.6/subprocess.py", line 369, in check_returncode
    self.stderr)
subprocess.CalledProcessError: Command '['/tmpfs/src/github/synthtool/env/bin/python3', '-m', 'synthtool', '--metadata', 'synth.metadata', 'synth.py', '--']' returned non-zero exit status 1.

```

Google internal developers can see the full log [here](http://sponge2/results/invocations/f3d0a9ba-acec-4587-9212-8f32deca2034/targets/github%2Fsynthtool;config=default/tests;query=google-cloud-ruby;failed=false).

--- a/google-cloud-managed_identities/synth.py
+++ b/google-cloud-managed_identities/synth.py
@@ -29,7 +29,7 @@ library = gapic.ruby_library(
         "ruby-cloud-title": "Managed Service for Microsoft Active Directory API",
         "ruby-cloud-description": "The Managed Service for Microsoft Active Directory API is used for managing a highly available, hardened service running Microsoft Active Directory.",
         "ruby-cloud-env-prefix": "MANAGED_IDENTITIES",
-        "ruby-cloud-wrapper-of": "v1:0.3,
+        "ruby-cloud-wrapper-of": "v1:0.3",
         "ruby-cloud-product-url": "https://cloud.google.com/managed-microsoft-ad/",
         "ruby-cloud-api-id": "managedidentities.googleapis.com",
         "ruby-cloud-api-shortname": "managedidentities",

